pkg/client/results/reader.go

```diff
@@ -53,6 +53,10 @@ const (
 	defaultNodesFile          = "Nodes.json"
 	defaultServerVersionFile  = "serverversion.json"
 	defaultServerGroupsFile   = "servergroups.json"
+
+	// InfoFile contains data not that isn't strictly in another location
+	// but still relevent to post-processing or understanding the run in some way.
+	InfoFile = "info.json"
```

Comment:
Given that a method is provided to return the info file path, does it need to be exported?

Reply:
So you raise a good question which I definitley touched on when coding this: who is in charge of what information and what is the right way to grab it?

A few different packages hardcode these string values (results.json, errors.json, results/, etc), some of the values here are exported, some arent. Some have methods to return their values, some dont.

I'm going to make a new ticket specifically for fixing this up and clarifying where to find all the results structure (folders/names/etc) and how to use them.

Reply:
https://github.com/heptio/sonobuoy/issues/912

TIL: there is a quick 'Reference comment in new issue' button which will create a new issue with the comment body.